### PR TITLE
rename invoke to request

### LINKED ISSUE #550: Rename invoke in JS SDK
'send' or 'request' seems like a better names. Invoke is an outdated name, since JS SDK can't specify invoked method anymore.

### PR DESCRIPTION
**Pull request by DieMyst**
fixes #550

### CODE REVIEW

File: fluence-js/src/ResultAwait.ts [hunk 1]
```diff
@@ -155,12 +152,6 @@ export class ResultAwait implements ResultPromise {
 
             detailedDebug("result is empty");
 
-            // here the session is checked after the result is checked, as it may happen that the result is given,
-            // and after that the session was immediately closed
-            if (sessionInfo.exists(isActive)) {
```
Comment (folex) [inline fluence-js/src/ResultAwait.ts]: Why remove this logic?

File: fluence-js/src/Session.ts [hunk 2]
```diff
@@ -129,20 +129,9 @@ export class Session {
...
+     * Syncs on all pending requests.
      */
     async sync(): Promise<Result> {
         return this.lastResult.result();
     }
-
-    /**
-     * Closes session locally and send a command to close the session on the cluster.
-     */
-    close(reason: string = ""): ResultPromise {
```
Comment (folex) [inline fluence-js/src/Session.ts]: Why remove this?

**Comment by folex** [comment]
Also, update this in tutorials

File: fluence-js/src/ResultAwait.ts (hunk 1 above)
Comment (DieMyst) [reply fluence-js/src/ResultAwait.ts]: this is not working on backend anymore.

**Pull request merged**